api.md

```diff
@@ -3406,138 +3312,143 @@ SELECT pg_size_pretty(after_compression_total_bytes) as total
 total | 48 kB
 
 ```
-## hypertable_relation_size() [](hypertable_relation_size)
+---
+
+## hypertable_detailed_size() :community_function: [](hypertable_detailed_size)
 
-Get relation size of hypertable like `pg_relation_size(hypertable)`.
+Get size of hypertable like `pg_relation_size(hypertable)`.
+If this is a distributed hypertable, the function returns size
+information per node. All sizes are reported in bytes.
 
-#### Required Arguments [](hypertable_relation_size-required-arguments)
+#### Required Arguments [](hypertable_detailed_size-required-arguments)
 
 |Name|Description|
 |---|---|
 | `main_table` | Identifier of hypertable to get relation size for.|
```

Comment:
Why isn't the name of the argument "hypertable"?

Also, in the chunks_* docs, it describes this as:  "Name of the hypertable", but here as "Identifier of hypertable to get relation size for."

Should this just be "Name of the hypertable"?

It can be confusing if we describe things differently even though we mean the same thing.  (Also, "identifier" here is slightly odd choice of language.)

(Same question for other functions here.)